### ISSUE TITLE
[Question] Worse output (even on train data) with wierd loss curve for fully finetuned model.

### ISSUE THREAD

**Issue by ninjacode01** [haotian-liu/LLaVA#1117]
### Question

**Gibberish output (even on train data) with wierd loss curve on fully finetuning. Can someone please help me fix this.**

I am trying to fully finetune the entire text-only model Vicuna-v1.5 using my custom QnA data comprising of 160k qa pairs, using the same finetuning script as provided in [finetune_task.sh](https://github.com/haotian-liu/LLaVA/blob/main/scripts/v1_5/finetune_task.sh) by omitting the multimodal parameters.
Here is the loss curve on 2.4 epochs. 
[wandb report](https://api.wandb.ai/links/hwadhwa-iitd/yd00o7ib
)![image](https://github.com/haotian-liu/LLaVA/assets/77221698/29a1c6fa-c944-4ecc-bd19-02e61ed3e09f)
